sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -10169,6 +10169,70 @@
               "id": "en74evs",
               "defaultValue": ""
             },
+            {
+              "clearOnHide": false,
+              "key": "everInCustodyOfChildWelfareInformationPanel",
```

Comment:
Does `ever` here can potentially imply "always" which would change the meaning of the variable name?
Maybe just `custodyOfChildWelfareInformationPanel` would be enough.

Reply:
update the key for new subquestion as well